api/src/main/java/org/pmiops/workbench/access/AccessModuleService.java

```diff
@@ -6,15 +6,17 @@
 import org.pmiops.workbench.db.model.DbAccessModule.DbAccessModuleName;
 import org.pmiops.workbench.db.model.DbUser;
 import org.pmiops.workbench.model.AccessBypassRequest;
-import org.pmiops.workbench.model.AccessModule;
 import org.pmiops.workbench.model.AccessModuleStatus;
 
 public interface AccessModuleService {
   /** Updates bypass time for a module. */
   void updateBypassTime(long userId, AccessBypassRequest accessBypassRequest);
 
   /** Updates bypass time for a module. */
-  void updateBypassTime(long userId, AccessModule accessModuleName, boolean isBypassed);
+  void updateBypassTime(long userId, DbAccessModuleName accessModuleName, boolean isBypassed);
```

Comment:
Changed the signature here to make it more consistent with similar calls and because more callers now have a `DbAccessModuleName` rather than an `AccessModule`